src/core.js

```diff
@@ -240,6 +244,10 @@ function resumeProcessing( test ) {
 
 			begin();
 		}, 13 );
+
+		if ( test ) {
+			test.resuming = resume;
```

Comment:
I'd prefer a boolean—and per the above, it's probably `test.resumed` (this property doesn't leak to consumers, does it?).